### DISCUSSION TITLE
how to set secondary electron emit

### DISCUSSION THREAD

**Discussion by wphu** [BLAST-WarpX/warpx#4334] [Q&A] [answered]
Hello,

I want to simulate glow discharge plasma, and "secondary electron emit" should be set. When the ion impacts on the cathode, a electron will emit from cathode surface with a certain probability.
could someone tell me how to do it?

thank you very much.

**Comment by RemiLehe** [comment, marked as answer]
Hi @wphu 
This functionality is not yet available in WarpX, but we are actively discussing how to add it in the code. I could give you an update on this in a few months from now.

**Reply by budjensen** [reply]
Hi @RemiLehe, is there any update about the status of adding secondary electron emission as a boundary condition?

**Comment by wphu** [comment]
> Hi @wphu This functionality is not yet available in WarpX, but we are actively discussing how to add it in the code. I could give you an update on this in a few months from now.

I am very much looking forward to this update， thank you.
